src/Components/CreateImageWizardV2/validators.ts

```diff
@@ -8,3 +8,13 @@ export const isAwsAccountIdValid = (awsAccountId: string | undefined) => {
   }
   return false;
 };
+
+export const isGcpEmailValid = (gcpShareWithAccount: string | undefined) => {
+  if (
+    gcpShareWithAccount !== undefined &&
+    /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,253}$/.test(gcpShareWithAccount)
+  ) {
+    return true;
+  }
+  return false;
```

Comment:
Slight code improvement:

```
return gcpShareWithAccount !== undefined && 
    /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,253}$/.test(gcpShareWithAccount)
```

does the same as:
```
  if (
    gcpShareWithAccount !== undefined &&
    /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,253}$/.test(gcpShareWithAccount)
  ) {
    return true;
  }
  return false;
```

but shorter :slightly_smiling_face: